coding/unit_test_policy.rst

```diff
@@ -79,7 +79,7 @@ Test suite execution should be managed by a testing framework, also known as a
 test harness, which monitors the execution status of individual test cases.
 
 C++: boost.test
-    LSST DM developers should use the `single-header variant`_ of the `Boost Unit
+    LSST DM developers should use the `shared-header variant`_ of the `Boost Unit
```

Comment:
Should be shared library, not shared header (there's no such thing). Otherwise looks good.